src/ZfcDatagrid/Column/AbstractColumn.php

```diff
@@ -602,39 +613,81 @@ public function getRendererParameters($rendererName = 'jqGrid')
     }
 
     /**
-     * Set a a template formatter
+     * Set a template formatter and overwrite other formatter
+     *
+     * @param AbstractFormatter[] $formatters
+     */
+    public function setFormatters(array $formatters)
+    {
+        $this->formatters = $formatters;
+    }
+
+    /**
+     * Set a template formatter and overwrite other formatter
      *
      * @param AbstractFormatter $formatter
+     * @deprecated please use setFormatters
      */
     public function setFormatter(AbstractFormatter $formatter)
     {
-        $this->formatter = $formatter;
+        trigger_error('Please use setFormatters()', E_DEPRECATED);
```

Comment:
E_DEPRECATED
will generate in ErrorLog:
PHP Warning:  Invalid error type specified in ......ZfcDatagrid/Column/AbstractColumn.php on line 633

i think its better to use
E_USER_DEPRECATED
will generate in ErrorLog:
PHP Deprecated:  Please use setFormatters() in 

check also http://php.net/manual/en/errorfunc.constants.php